doc/api/GVT-g_api.rst

```diff
@@ -0,0 +1,114 @@
+.. _GVT-g_api:
+
+===============
+ACRN GVT-g APIs
+===============
+
+GVT-g is the GPU virtualization solution of Intel, it is open source and
+up-streamed to the Linux kernel. Its implementation over KVM is named KVMGT,
+over Xen is named XenGT and over ACRN is named ACRN-GT. GVT-g could exports
```

Comment:
add a comma:  ``... named XenGT, and over ACRN ...

Replace the next sentence with:

    GVT-g can export multiple virtual GPU (vGPU) instances for
    virtual machine (VM) systems.

Reply:
We should also add a couple of  subject ('it), i.e.:
...over Xen it is named XenGT, and over ACRN it is named ACRN-GT

A different question, it's the first time I see the term ACRN-GT coined, any reason why we're not using ACRNGT (all attached, just like for Xen and KVM)? (I personnally like it better with a hyphen but it's a divergence from the other implementations)